scipy/optimize/_qap.py

```diff
@@ -43,97 +41,100 @@ def quadratic_assignment(
-                barycenter will likely result in a more accurate result at
-                the cost of higher runtime.
-
-                The initial position chosen:
-
-                "barycenter" : the non-informative "flat doubly stochastic
-                matrix," :math:`J=1*1^T /n` , i.e the barycenter of the
-                feasible region (where :math:`n` is the number of nodes and
-                :math:`1` is a ``(n, 1)`` array of ones).
-
-                If an ndarray is passed, it should have the same shape as
-                `cost_matrix` and `dist_matrix`, and its rows and columns
-                must sum to 1 (doubly stochastic).
-            init_weight : float, positive in range [0,1] (Default = None)
-                Allows the user to specify the amount of random perturbation
-                from the starting position of `init`
-                At each initialization, the initial permutation matrix
-                is some random point near :math:`J`, defined as
-                :math:`(\alpha J + (1- \alpha) K`, where :math:`\alpha`
-                is `init_weight` and :math:`K` is some random doubly
-                stochastic matrix.
-            init_n : int, positive (default = 1)
-                Number of random initializations of the starting
-                permutation matrix that the FAQ algorithm will undergo.
+        A dictionary of solver options.
+
+            init_J : 2d-array or "barycenter" (default = "barycenter")
+                The initial (guess) permutation matrix or search "position"
+                :math:`J`.
+
+                :math:`J` need not be a proper permutation matrix;
+                however, it must have the same shape as `cost_matrix` and
+                `dist_matrix`, and it must be doubly stochastic: each of its
+                rows and columns must sum to 1.
+
+                If unspecified or ``"barycenter"``, the non-informative "flat
+                doubly stochastic matrix" :math:`1*1^T/n`, where :math:`n`
+                is the number of nodes and :math:`1` is a :math:`n \times 1`
+                array of ones, is used. This is the "barycenter" of the
+                search space of doubly-stochastic matrices.
+            init_weight : float in range [0, 1] (default = 1)
+                Allows the user to specify the weight of the provided
+                search position :math:`J` relative to random perturbations
+                :math:`K`.
+
+                The algorithm will be repeated :math:`k` times
+                from randomized initial search positions
+                :math:`P_0 = (\alpha J + (1- \alpha) K`, where
+                :math:`J` is given by option `init_J`,
+                :math:`\alpha` is given by option `init_weight`,
+                :math:`k` is given by option `init_k`, and
+                :math:`K` is a random doubly stochastic matrix.
+            init_k : int, positive (default = 1)
```

Comment:
I changed this to `init_k` because `n` was used as the number of nodes. There are `n` nodes and `k` random perturbation matrices `K`.